src/sdk/bucketspace_fs.js

```diff
@@ -58,6 +58,8 @@ class BucketSpaceFS extends BucketSpaceSimpleFS {
             uid: process.getuid(),
             gid: process.getgid(),
             warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
+            fs_backend: config.NSFS_NC_CONFIG_DIR_BACKEND
+            //fs_context.report_fs_stats = this.stats.update_fs_stats;
```

Comment:
please remove the commented code

Reply:
we still need to support it I just copied it from line 44 as a reminder